subscribers.js

```diff
@@ -1,7 +1,89 @@
 "use strict";
 
-// Set of all registered emails.
-// TODO: Implement a real persistent storage solution.
-const gEmails = new Set();
+const pg = require("pg");
 
-module.exports = gEmails;
+const dbConfig = {
+  max: 10,
```

Comment:
Is this something that OPs or whoever may want to tweak?

Reply:
Hmm, this would be nice, yes, but I'm going to procrastinate on it because:
- This is not immediately important.
- Getting numerical values from the environment means using parseint, and that seems like a prime surface area for bugs and failures. 
- We should probably first improve AppConstants to also know what type each value is supposed to be and automate the parseints.

Reply:
Cool. Filed as #62 for tracking.